### ISSUE TITLE
use int64

### ISSUE THREAD

**Issue by tcoupin** [yashbhutwala/kubectl-df-pv#7]
this tool is very useful! 

https://github.com/yashbhutwala/kubectl-df-pv/blob/5813f63ec9d172421374c5519f28d409954717f5/pkg/plugin/plugin.go#L84

=> response: int64 :)

```
> kubectl df-pv
  Searching for PVCs ⠴json: cannot unmarshal number 18446744073709551615 into Go struct field Volume.inodesFree of type int
```

**Comment by yashbhutwala** [comment]
Fixed.  I'm glad you find it useful! 🎉  Thanks for taking the time to create the issue! 💯

**Comment by tcoupin** [comment]
int64 is not enough yet, use uint64

```
> kubectl df-pv
E0713 08:35:47.140602   59740 request.go:975] Unexpected error when reading response body: context canceled
Error: error getting output slice: failed to convert the response from server: json: cannot unmarshal number 18446744073709551615 into Go struct field Volume.pods.volume.inodesFree of type int64
Usage:
  df-pv [flags]

Flags:
  -h, --help               help for df-pv
  -n, --namespace string   if present, the namespace scope for this CLI request (default is all namespaces)
  -v, --verbosity string   log level; one of [info, debug, trace, warn, error, fatal, panic] (default "info")

FATA[2020-07-13T08:35:47+02:00] unable to run root command: json: cannot unmarshal number 18446744073709551615 into Go struct field Volume.pods.volume.inodesFree of type int64
failed to convert the response from server
github.com/yashbhutwala/kubectl-df-pv/pkg/df-pv.GetOutputRowPVCFromNodeChan
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/pkg/df-pv/root.go:523
github.com/yashbhutwala/kubectl-df-pv/pkg/df-pv.GetSliceOfOutputRowPVC.func2
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/pkg/df-pv/root.go:479
golang.org/x/sync/errgroup.(*Group).Go.func1
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/vendor/golang.org/x/sync/errgroup/errgroup.go:57
runtime.goexit
	/opt/hostedtoolcache/go/1.14.4/x64/src/runtime/asm_amd64.s:1373
error getting output slice
github.com/yashbhutwala/kubectl-df-pv/pkg/df-pv.runRootCommand
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/pkg/df-pv/root.go:72
github.com/yashbhutwala/kubectl-df-pv/pkg/df-pv.setupRootCommand.func1
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/pkg/df-pv/root.go:53
github.com/spf13/cobra.(*Command).execute
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/vendor/github.com/spf13/cobra/command.go:842
github.com/spf13/cobra.(*Command).ExecuteC
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/vendor/github.com/spf13/cobra/command.go:950
github.com/spf13/cobra.(*Command).Execute
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/vendor/github.com/spf13/cobra/command.go:887
github.com/yashbhutwala/kubectl-df-pv/pkg/df-pv.InitAndExecute
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/pkg/df-pv/root.go:30
main.main
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/cmd/df-pv/main.go:10
runtime.main
	/opt/hostedtoolcache/go/1.14.4/x64/src/runtime/proc.go:203
runtime.goexit
	/opt/hostedtoolcache/go/1.14.4/x64/src/runtime/asm_amd64.s:1373
run df-pv root command
github.com/yashbhutwala/kubectl-df-pv/pkg/df-pv.InitAndExecute
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/pkg/df-pv/root.go:30
main.main
	/home/runner/work/kubectl-df-pv/kubectl-df-pv/cmd/df-pv/main.go:10
runtime.main
	/opt/hostedtoolcache/go/1.14.4/x64/src/runtime/proc.go:203
runtime.goexit
	/opt/hostedtoolcache/go/1.14.4/x64/src/runtime/asm_amd64.s:1373
```

**Comment by yashbhutwala** [comment]
@tcoupin thanks again for reporting!  I fixed it via commit: https://github.com/yashbhutwala/kubectl-df-pv/commit/275860eba54fd66bbf33866d1b6cd513fb07c072 and it will be released at `v0.2.4` shortly. 

Interesting thing is `resource.Quantity` is used for all the `inodes`, and `resource.Quantity` only takes a int64 as `NewQuantity(value int64, format Format)`, so I wonder if this is a k8s bug as well for the metrics...
